### PR TITLE
add a scripts to make trafodion build easily and Visualized

### PR DESCRIPTION
**Pull request by CoderSong2015**
There are several unpleasant places when compiling the trafodion(not make package):
1. We need to redirect the make information manually,
   otherwise the information will flood the screen
2. The build information mixed together so that we couldn't locate the build error quickly
3. We need to make all again when we fix a building error of one component.

This scripts could build trafodion component based.
And it will write the compiling information into build_log file.
Also you can continue to compile the remaining components which havn't been compiled by commented components which have finished compiled.

### CODE REVIEW

**Comment by mkby** [comment]
Super cool! Can we also add 'make package'?

**Comment by CoderSong2015** [comment]
> Super cool! Can we also add 'make package'?

Yes,. It is better to do it in next Pr.

File: build_easily.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```
Comment (xiaozhongwang) [inline build_easily.sh]: need the copyright header here

File: build_easily.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+fi
+
+if [ ! -d $log_dir/"foundation" ];then
+    mkdir $log_dir/"foundation"
+fi
+
+#export LC_ALL=en_US.UTF-8
+cd $TRAF_HOME/../
+./bldenvchk.sh;
+export SQ_BUILD_TYPE=debug
```
Comment (xiaozhongwang) [inline build_easily.sh]: you can use debug as default, and developer can change it via parameter.

**Comment by ryzuo** [comment]
Finally! This looks great!

**Comment by weiqxu** [comment]
It's an amazing work.

File: build_easily.sh (hunk 2 above)
Comment (CoderSong2015) [reply build_easily.sh]: Hi, this srcipt now can olny be used in debug env.

**Comment by CoderSong2015** [comment]
Can anyone make a review？

**Comment by CoderSong2015** [comment]
> I think building via this script will be slower, since it is not doing things with as much parallelism as make would. It also has potential to get out of synch with makefiles / dependencies. So I would not document it as the primary build mechanism, but it seems fine as an alternative for those that want to use it.

@svarnau The script just call the make of the trafodion components so that it will not build slower than executing make directly .Besides, the dependency between components make that impossible to build component in parallel.

**Pull request merged**